File: ModuleService/src/main/java/cn/mycommons/moduleservice/IOrderService.java
package cn.mycommons.moduleservice;

import java.util.List;

import cn.mycommons.moduleservice.bean.OrderInfo;

/**
 * IOrderService <br/>
 * Created by xiaqiulei on 2017-05-14.
 */
public interface IOrderService {

    List<OrderInfo> getAllOrders();
}

File: ModuleService/src/main/java/cn/mycommons/moduleservice/IOrderService.kt
package cn.mycommons.moduleservice

import cn.mycommons.moduleservice.bean.OrderInfo

/**
 * IOrderService <br></br>
 * Created by xiaqiulei on 2017-05-14.
 */
interface IOrderService {

    fun getAllOrders(): List<OrderInfo>
}